<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Gradent Export</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_select_all">
       <property name="text">
        <string>Select All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_deselect_all">
       <property name="text">
        <string>Deselect All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="gradient_list">
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
     <property name="currentRow">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Item1</string>
      </property>
      <property name="checkState">
       <enum>PartiallyChecked</enum>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Item2</string>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="save_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Save Selected</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
